code/range/Model/src/main/java/ru/unn/agile/range/model/Range.java

```diff
@@ -0,0 +1,79 @@
+package ru.unn.agile.range.model;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public class Range {
+    private int startingElement;
+    private int finiteElement;
+
+    public Range(final String rangeString) {
+        String trimRangeString = rangeString.trim();
+        String[] sentences = trimRangeString.split("[\\[(,\\s\\])]+");
+        startingElement = Integer.parseInt(sentences[1]);
+        finiteElement = Integer.parseInt(sentences[2]);
+        if (trimRangeString.startsWith("(")) {
+            startingElement++;
+        }
+        if (trimRangeString.endsWith(")")) {
+            finiteElement--;
+        }
+        if (startingElement > finiteElement) {
+            throw new RuntimeException("No numbers in the given interval!");
```

Comment:
Why RuntimeException? IllegalArgumentException in that case is more appropriate.

Reply:
Ok